src/sys/event.rs

```diff
@@ -79,16 +78,18 @@ pub enum EventFilter {
 }
 
 #[cfg(any(target_os = "macos", target_os = "ios",
-          target_os = "freebsd", target_os = "dragonfly"))]
+          target_os = "freebsd", target_os = "dragonfly",
+          target_os = "openbsd"))]
 pub type type_of_event_flag = u16;
-#[cfg(any(target_os = "netbsd", target_os = "openbsd"))]
+#[cfg(any(target_os = "netbsd"))]
 pub type type_of_event_flag = u32;
 libc_bitflags!{
     pub flags EventFlag: type_of_event_flag {
         EV_ADD,
         EV_CLEAR,
         EV_DELETE,
         EV_DISABLE,
+        #[cfg(not(target_os = "openbsd"))]
```

Comment:
By the looks of it they aren't in a released version of OpenBSD yet. The author of that patch added it to the current branch but there hasn't been a release since. I think the linked PR is still unmerged as a result.

I think the right thing to do for now is to leave it out. I can raise another issue to add it when the next OpenBSD release occurs.